app/rc_client.py

```diff
@@ -29,35 +36,39 @@ def send_speed(self, speed, angular_speed):
         speed = math.copysign(min(self.max_spedd, abs(speed)), speed)
         angular_speed = math.copysign(min(self.max_angular_speed, abs(angular_speed)), angular_speed)
         self.bus.publish('desired_speed', [round(speed * 1000), round(math.degrees(angular_speed) * 100)])
-        print(speed, angular_speed)
-
-        return speed, angular_speed
 
 
     def run_input(self):
         speed = 0
         angular_speed = 0
+        angel = 0
```

Comment:
angle? (typo) ... angel is nicer :) ... I would maybe also use units `angle_rad`?